api/services/itemsService.js

```diff
@@ -0,0 +1,12 @@
+require("dotenv").config();
+const itemsDao = require("../models/itemsDao");
+
+
+const getItems = async (itemsId, options) => {
+  const { results, results2 } = await itemsDao.getItems(itemsId, options);
+  return { results, results2 };
```

Comment:
model에 드린 리뷰를 적용 하시면 다음과 같이 수정해 주시면 될 것 같습니다.
```suggestion
  return  await itemsDao.getItems(itemsId, options);
```